exemples/gt_non_recours/README.md

```diff
@@ -0,0 +1,69 @@
+## Supports du Groupe de Travail de lutte contre le non recours
+  
+ > Supports du GT `Accès aux droits et aux services, lutte contre le non recours`  
+ de la `Délégation Interministérielle à la prévention et à la lutte contre la pauvreté des enfants et des jeunes`  
+ Le rapport de ce Groupe de Travail est accessible [ici](https://www.caissedesdepotsdesterritoires.fr/cs/BlobServer?blobkey=id&blobnocache=true&blobwhere=1250171076233&blobheader=application%2Fpdf&blobcol=urldata&blobtable=MungoBlobs).
+
+### Description du contenu
+
+* Le script `framework_no_reform/investiguer_achetypes.py` execute un ensemble de calculs sur une ou plusieurs periodes et sur les situations contenues dans `/situations`.  
+Les resultats sont repertoriés dans un fichier `csv`.
+
```

Comment:
This file seems to not be in this directory anymore ...

Reply:
investiguer_achetypes.py --> investiguer_archetypes.py

Reply:
Moved into new `python/README.md`.